task-2/src/components/SearchBar/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+
+import movieData from '../Movie/data.js'
+import './index.css'; 
+
+export const SearchBar = () => {
+    const [searchTerm, setSearchTerm] = useState('')
+
+    const filterHandler = (value) => {
+        if (searchTerm === "") {
+            return value
```

Comment:
Looks like we're returning the same value "value" here in every case which makes the "if statement" redundant.
Consider refactoring this logic if possible.

Reply:
Logic for searchbar component has been removed - waiting for state implementation.

All components have been changed to ES6 syntax.